.changeset/tiny-worms-drum.md

```diff
@@ -0,0 +1,6 @@
+---
+"@marigold/docs": patch
```

Comment:
This one is breaking too :D Guess we should include this in the next release

Reply:
oh sorry yes